src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -221,13 +232,35 @@ public void writeThrough(Conversation conversation) {
   }
 
   /** Write a Media object to the Datastore service. */
-  public void writeThrough(Media singleMedia) {
+  public void writeThrough(Media singleMedia) throws java.io.IOException {
     Entity mediaEntity = new Entity("chat-media", singleMedia.getId().toString());
     mediaEntity.setProperty("uuid", singleMedia.getId().toString());
     mediaEntity.setProperty("owner_uuid", singleMedia.getOwnerId().toString());
     mediaEntity.setProperty("title", singleMedia.getTitle());
     mediaEntity.setProperty("creation_time", singleMedia.getCreationTime().toString());
-    mediaEntity.setProperty("content", singleMedia.getContent().toString());
+    
+    ByteArrayOutputStream baos = new ByteArrayOutputStream();
+    ImageIO.write(singleMedia.getContent(), "jpg", baos);
```

Comment:
Again, you are assuming the content type is image here.
But in the future, if we handle other types of content. We need to have conditional that checks on the type of the content then behavior differently, e.g., ImageIO, maybe AudioIO, etc?